bin/sbang

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env sh
```

Comment:
`sbang` is vendored and while I don't see a reason why this won't work in general -- why do this?  Is it *just* so that we can use a shell built by Spack?